enhancements/windows-containers/windows-machine-config-operator.md

```diff
@@ -95,16 +96,18 @@ secret with a public key that corresponds to the private key. In addition it
 will have all the pre-requisite steps needed for setting the VM up for key based
 SSH connections.
 
-When the cluster admin wants to bring up Windows worker nodes, they will create
-a [MachineSet](https://github.com/openshift/machine-api-operator/blob/master/config/machineset.yaml)
-where they reference a Windows OS image which has the Docker container runtime
-add-on enabled. They will also specify the user data secret created by WMCO as
+When the cluster admin is using a WMCO version in the range 1.x-5.x 
+to bring up Windows worker nodes, they will create a [MachineSet](https://github.com/openshift/machine-api-operator/blob/master/config/machineset.yaml) where they reference a Windows OS image which has the Docker 
```

Comment:
`they will create MachineSets...`

Please format with 80 columns. It is okay for a the URL to go beyond 80 columns.